modules/shadertools/src/modules/pbr/pbr-fragment.glsl.js

```diff
@@ -312,12 +328,14 @@ vec4 pbr_filterColor(vec4 colorUnused)
   // This section uses mix to override final color for reference app visualization
   // of various parameters in the lighting equation.
 #ifdef PBR_DEBUG
-  color = mix(color, F, u_ScaleFGDSpec.x);
-  color = mix(color, vec3(G), u_ScaleFGDSpec.y);
-  color = mix(color, vec3(D), u_ScaleFGDSpec.z);
-  color = mix(color, specContrib, u_ScaleFGDSpec.w);
+  // TODO: Figure out how to debug multiple lights
```

Comment:
Maybe in debug mode restrict to the first light?

Reply:
I though I should combine all